docs/data-sources/aws_unity_catalog_assume_role_policy.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Deployment"
+---
+# databricks_aws_unity_catalog_assume_role_policy Data Source
+
+-> **Note** This resource has an evolving API, which may change in future versions of the provider. Please always consult [latest documentation](https://docs.databricks.com/data-governance/unity-catalog/get-started.html#configure-a-storage-bucket-and-iam-role-in-aws) in case of any questions.
+
+This data source constructs necessary AWS Unity Catalog assume role policy for you. [official documentation]
+
+## Example Usage
+
+```hcl
+data "databricks_aws_unity_catalog_policy" "this" {
+  aws_account_id = var.aws_account_id
+  bucket_name    = "databricks-bucket"
+  role_name      = "${var.prefix}-uc-access"
+  kms_name       = "databricks-kms"
```

Comment:
There is no such parameter in the struct

Reply:
this is for `databricks_aws_unity_catalog_policy`